Compiler/test2/cc_bytecode_test_1.cpp

```diff
@@ -1548,7 +1549,7 @@ TEST_F(Bytecode1, Attributes08) {
 TEST_F(Bytecode1, Attributes09) {
 
     // Function call to 'set_Visible()' mustn't go awry
-    // After loading 'true' to AX�, must protect AX from being clobbered (push it)
+    // After loading 'true' to AX�, must protect AX from being clobbered (push it)
```

Comment:
Another case of strange symbols, maybe automatic fix by a visual studio?

Reply:
this file is not encoded in UTF-8 and is instead using some other encoding I can't figure it out...

Reply:
fixed by removing this symbol